libs/shared/profile-form/src/lib/shared-profile-form.scss

```diff
@@ -1,112 +1,164 @@
-.profileForm {
-  min-width: 700px;
-  padding-bottom: 15px;
-  .MuiBox-root-11,
-  .MuiBox-root-59 {
-    margin-top: 0;
+.profile-form-container {
+  padding: 30px;
+  .header {
+    font-size: 18px;
+    margin-bottom: 4px;
   }
-  .text-row {
+  .draggable-container.empty {
+    min-height: 6em;
+  }
+
+  .draggable-container {
+    min-width: 100%;
+    min-height: 6em;
+    padding: 0px;
+    border-radius: 4px;
+    border: none;
+    font-size: 14px;
     display: flex;
-    justify-content: space-between;
-    .firstName,
-    .lastName,
-    .email,
-    .phone,
-    .website,
-    .type,
-    .facebook,
-    .instagram,
-    .twitter,
-    .linkedin {
+    margin-bottom: 7px;
+    position: relative;
+    overflow: hidden;
+
+    #file-browser-input,
+    & > div,
+    & > div > div {
+      background-color: transparent;
+      cursor: pointer;
       width: 100%;
+      height: 100%;
+      top: 0px;
+      left: 0px;
+      position: absolute;
+      opacity: 1;
+      z-index: 100;
     }
-  }
 
-  .text-column {
-    width: 48%;
-    display: flex;
-    flex-direction: column;
-  }
-  .text-columns {
-    width: 32%;
-    display: flex;
-    flex-direction: column;
-  }
+    #file-browser-input {
+      opacity: 0;
+    }
 
-  .select-button {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    // margin-bottom: 8.5px;
-  }
-  .submit-form {
-    margin-top: 0.5rem !important;
-    padding: 0.5rem !important;
-    width: 48%;
-  }
+    & > div > div:before,
+    & > div > div:after {
+      content: none !important;
```

Comment:
I would prefer to avoid !important if possible. Write full class/elements path instead.